src/super_gradients/modules/conv_bn_act_block.py

```diff
@@ -67,6 +68,14 @@ def __init__(
     def forward(self, x):
         return self.seq(x)
 
+    def replace_input_channels(self, in_channels: int, compute_new_weights_fn: Optional[Callable[[nn.Module, int], nn.Module]] = None):
+        from super_gradients.modules.weight_replacement_utils import replace_conv2d_input_channels
+
+        self.seq[0] = replace_conv2d_input_channels(conv=self.seq[0], in_channels=in_channels, fn=compute_new_weights_fn)
+
+    def get_input_channels(self) -> int:
+        return self.seq[0].in_channels
+
 
 class Conv(nn.Module):
```

Comment:
Missing `, SupportsReplaceInputChannels` mixin here?

Reply:
I'm confused, it looks like it does inherit from it: `class ConvBNAct(nn.Module, SupportsReplaceInputChannels):`
Or are you talking about something else?

Reply:
He's talking of class Conv(nn.Module):

Reply:
oh

Reply:
[503129f](https://github.com/Deci-AI/super-gradients/pull/1557/commits/503129f8cdee297a3c555316f481e7a4502ca62c)